reana_client/cli/files.py

```diff
@@ -79,8 +80,9 @@ def get_files(
     `--workflow` or `-w`.
 
     Examples: \n
-    \t $ reana-client ls --workflow myanalysis.42
-    """
+    \t $ reana-client ls --workflow myanalysis.42 \n
+    \t $ reana-client ls --workflow myanalysis.42 'code/\*'
```

Comment:
```
reana ❯ reana-client ls --help
....
  Examples:
       $ reana-client ls --workflow myanalysis.42 'code/\*'
```
`'code/\*'` is printed to user which is not valid example

Reply:
The wildcard needs to be escaped, otherwise the readthedocs build fails as it interprets `*` as the beginning of a word in bold. More info [here](https://stackoverflow.com/questions/30454549/a-literal-in-restructuredtext). This is very unfortunate as the example is not working then.. But I can't find another solution. OTOH, it's displayed properly in RTD sit. We have the same in `reana-client rm`.